js/main.js

```diff
@@ -23,4 +85,54 @@ const getRandomFloatInclusive = function (min, max, fix = 2) {
   return -1;
 }
 
-getRandomFloatInclusive (1, 5, 3);
+
+const getRandomArrayElement = (elements) => {
+  const indexElement = getRandomIntInclusive(0, elements.length-1)
+  return elements[indexElement];
+};
+
+const getUniqueArray = (array, length) => {
+  const uniqueArray = [];
+  for (let i=0; i <= length-1; i++) {
+    uniqueArray.push(getRandomArrayElement)
+  }
+  return uniqueArray;
+};
+
+const getLocation = () => {
+  const x = getRandomFloatInclusive (xMinLocation, xMaxLocation, 5);
```

Comment:
Перед скобками вызова ф-и не нужны пробелы
```suggestion
  const x = getRandomFloatInclusive(xMinLocation, xMaxLocation, 5);
```